velox/functions/sparksql/DecimalArithmetic.cpp

```diff
@@ -14,25 +14,16 @@
  * limitations under the License.
  */
 
-#include "velox/common/base/CheckedArithmetic.h"
 #include "velox/expression/DecodedArgs.h"
 #include "velox/expression/VectorFunction.h"
+#include "velox/functions/Macros.h"
+#include "velox/functions/Registerer.h"
 #include "velox/functions/sparksql/DecimalUtil.h"
 #include "velox/type/DecimalUtil.h"
 
 namespace facebook::velox::functions::sparksql {
 namespace {
```

Comment:
This file is long and it looks like most functions defined in the anonymous namespace are for decimal add and subtract. Wondering if it makes sense to separate add & subtract in one file, multiply in one file and divide in another.

Reply:
I would prefer decimal arithmetic functions to be one file. It is the similar feature. I will try if we can use a base class for add and subtract.